source/server/infrastructure/router.coffee

```diff
@@ -31,24 +32,38 @@ class Space.eventSourcing.Router extends Space.messaging.Controller
     if not @initializingCommand?
       throw new Error Router.ERRORS.missingInitializingCommand
     @routeCommands ?= []
+    @routeEvents ?= []
     super
 
   onDependenciesReady: ->
     super
+    @_setupInitializingCommand()
+    @_routeCommandToAggregate(commandType) for commandType in @routeCommands
+    @_routeEventToAggregate(eventType) for eventType in @routeEvents
```

Comment:
Hmm, not sure about routing events to Aggregates. Is this just to cover processes in the same class?